apps/meteor/app/ui-utils/client/lib/RoomHistoryManager.js

```diff
@@ -42,7 +42,7 @@ export const normalizeThreadMessage = ({ ...message }) => {
 
 export const waitUntilWrapperExists = async (selector = '.messages-box .wrapper') => {
 	const element = document.querySelector(selector);
-	if (element?.length) {
+	if (element) {
```

Comment:
I believe this will reintroduce the previous bug of not scrolling to the correct message when following a URL.

Reply:
maybe we need to test `typeof element`? I remember when first resolving the issue I mentioned, `document.querySelector` was bringing an "empty" element that was not falsy.